caikit_computer_vision/modules/text_to_image/sdxl_stub.py

```diff
@@ -0,0 +1,76 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for text to image via SDXL.
+"""
+# Standard
+from typing import Union, get_args
+import os
+
+# Third Party
+import numpy as np
+
+# First Party
+from caikit.core.modules import ModuleBase, ModuleConfig, ModuleSaver, module
+from caikit.interfaces.vision import data_model as caikit_dm
+import alog
+
+# Local
+from ...data_model import TextToImageResult
+from ...data_model.tasks import TextToImageTask
+
+log = alog.use_channel("SDXL")
+
+
+@module(
+    id="28aa938b-1a33-11a0-11a3-bb9c3b1cbb11",
+    name="Stub module for Text to Image",
+    version="0.1.0",
+    task=TextToImageTask,
+)
+class SDXLStub(ModuleBase):
+    def __init__(
+        self,
+        model_name,
+    ) -> "SDXLStub":
+        log.debug("STUB - initializing text to image instance")
+        super().__init__()
+        self.model_name = model_name
+
+    @classmethod
+    def load(cls, model_path: Union[str, "ModuleConfig"]) -> "SDXLStub":
+        config = ModuleConfig.load(model_path)
+        return cls.bootstrap(config.model_name)
+
+    @classmethod
+    def bootstrap(cls, model_name: str) -> "SDXLStub":
+        return cls(model_name)
+
+    def save(self, model_path: str):
+        saver = ModuleSaver(
+            self,
+            model_path=model_path,
+        )
+        with saver:
+            saver.update_config({"model_name": self.model_name})
+
+    def run(self, inputs: str, height: int, width: int) -> TextToImageResult:
```

Comment:
Ah, ok, so this seems to be _just_ a stub and not actually have anything to do with `sdxl`, right? I think we should change the name unless you plan to update it to actually support running `sdxl`. As written, I think this is more of a unit test stub, right?

Reply:
Ahh yes, sorry! This is a little disorganized - it is just a stub. I am still splitting some stuff apart a bit, but I wrote this so that software could test wiring more easily.

There *is* an SDXL block too, but it's in a separate PR that is on top of this one that I am rebasing at the moment ([here](https://github.com/caikit/caikit-computer-vision/pull/60))

Reply:
Ok, that makes sense. I still think this should be named `tti_stub` since we eventually imagine multiple modules for `tti` and this is not `sdxl` specific

Reply:
Agreed! Renamed